src/Microsoft.Extensions.Testing.Abstractions/TestOutcome.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Extensions.Testing.Abstractions
+{
+    public enum TestOutcome
```

Comment:
MSTest has a concept of `Inconclusive` as well. We might want to make sure we review the possible outcomes from various runners.